mmv1/third_party/terraform/tests/resource_dns_record_set_test.go.erb

```diff
@@ -11,23 +11,23 @@ import (
 )
 func TestIpv6AddressDiffSuppress(t *testing.T) {
 	cases := map[string]struct {
-		Old, New       string
+		Old, New       []string
 		ShouldSuppress bool
 	}{
 		"compact form should suppress diff": {
```

Comment:
you could also add the test case of multiple of the same ip address. the result is the diff should be suppressed.

Reply:
added